edit_here_to_add_blocklists.json

```diff
@@ -49,7 +49,7 @@
     {
       "name": "1Hosts (mini)",
       "category": "Ads, Privacy",
-      "url": ["https://raw.githubusercontent.com/badmojr/1Hosts/master/mini/domains.txt"],
+      "url": ["https://raw.githubusercontent.com/badmojr/1Hosts/master/mini/domains.wildcards"],
       "filterType": ["b-norm"],
```

Comment:
Here, filtertype should be `b-wild`.

Reply:
> Here, filtertype should be `b-wild`.

So I accidentally deleted a branch because it was bugged and it deleted this one along with it so it is not giving me the option to edit the file anymore. If you merge the commits. Il open a new pull request to change the filtertype